pkg/metricsservice/api/metrics.pb.go

```diff
@@ -16,7 +16,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.32.0
-// 	protoc        v4.23.2
+// 	protoc        v3.6.1
```

Comment:
Are you using outdated version of `protoc`? You shall not, undo changes to all the `.pb.go` files, please.

Reply:
I am using version 25.2. Latest version is 28.0. I am not sure if 25.2 is outdated.